spec/compiler/macro/macro_methods_spec.cr

```diff
@@ -276,12 +277,24 @@ module Crystal
         assert_macro "{{1e-123_f32.kind}}", ":f32"
         assert_macro "{{1.0.kind}}", ":f64"
         assert_macro "{{0xde7ec7ab1e_u64.kind}}", ":u64"
+        assert_macro "{{1_u128.kind}}", ":u128"
+        assert_macro "{{-20i128.kind}}", ":i128"
       end
 
       it "#to_number" do
         assert_macro "{{ 4_u8.to_number }}", "4"
         assert_macro "{{ 2147483648.to_number }}", "2147483648"
         assert_macro "{{ 1_f32.to_number }}", "1.0"
+        assert_macro "{{ 4_u128.to_number }}", "4"
+        assert_macro "{{ -20i128.to_number }}", "-20"
+      end
+
+      # Requires #11571
+      pending "executes math operations using U/Int128" do
```

Comment:
Just leaving this comment as a reminder to remove the `pending` after #11571

Reply:
done in c813d0550f68ea0f7ccdab0ff5d7fb1f7406cadc